react/components/LocaleSwitcherList.tsx

```diff
@@ -80,14 +80,15 @@ export default function LocaleSwitcherList(props: Props) {
             role="link"
             tabIndex={-1}
             key={localeId}
-            className={listElementClasses}
+            className={`${listElementClasses} ${handles.listElement}--${text}`}
```

Comment:
Hey @ygorneves10 👋 Check this example in the `vtex.css-handles` (https://github.com/vtex-apps/css-handles#usecsshandles) repo that shows how to use the `applyModifiers` to add this kind of classname modifier.